src/win32.mak

```diff
@@ -308,6 +321,8 @@ clean:
 	$(DEL) impcnvtab.c
 	$(DEL) id.h id.c
 	$(DEL) verstr.h
+	$(DEL) $(GENSRC)
+	$(DEL) $(MAGICPORT)
```

Comment:
It's only as terrible or as broken or as magically as the person who writes the makefile.